apps/utils/cipher/cipher.main.kts

```diff
@@ -6,23 +6,13 @@
 @file:DependsOn("com.squareup.okio:okio:3.5.0")
 
 import com.amazonaws.encryptionsdk.AwsCrypto
-import com.amazonaws.encryptionsdk.CryptoResult
-import com.amazonaws.encryptionsdk.kmssdkv2.KmsMasterKey
-import com.amazonaws.encryptionsdk.kmssdkv2.KmsMasterKeyProvider
+import com.amazonaws.encryptionsdk.kmssdkv2.AwsKmsMrkAwareMasterKeyProvider
 import kotlinx.collections.immutable.PersistentMap
 import kotlinx.collections.immutable.persistentMapOf
-import okio.Buffer
-import okio.BufferedSource
-import okio.ByteString
+import okio.*
```

Comment:
I'd really rather not allow what amounts to a style decision to hold this up any longer, but... maybe it's reversing this isn't so bad if you're using an IDE?